can/interfaces/slcan.py

```diff
@@ -103,7 +101,9 @@ def _recv_internal(self, timeout):
         remote = False
         extended = False
         frame = []
-        readStr = self.serialPort.readline()
+
+        readStr = self.serialPortOrig.read_until(b'\r')
+        readStr = readStr.decode('utf-8')
```

Comment:
Maybe this line should move after the `if not readStr:` check and inside the `else:` block?

Reply:
Done.